contrib/adaptive-compression/adapt.c

```diff
@@ -982,7 +1018,7 @@ static unsigned readU32FromChar(const char** stringPtr)
 static void help(void)
 {
     PRINT("Usage:\n");
-    PRINT("  ./multi [options] [file(s)]\n");
+    PRINT("  zstd-adaptive [options] [file(s)]\n");
```

Comment:
minor detail : 
another way to get the name of executable is to probe `argv[0]`.
Advantage is, it's dynamic.
Though it provides the full path, not just the binary name.